src/ducks/transfers/TransferPage.jsx

```diff
@@ -105,6 +108,10 @@ const NoBank = () => {
   )
 }
 
+const isMyselfSufficientlyFilled = myself => {
```

Comment:
This could be good to have in cozy-client models as we may need it elsewhere than in banks.

Reply:
Here, the sufficiently filled is dependent on the transfer function. I don't think it should be in the models yet. I'll try to keep in mind if it resurfaces elsewhere.